### DISCUSSION TITLE
eggs!

### DISCUSSION THREAD

**Discussion by lstebner** [jeremyckahn/farmhand#149] [Ideas]
we need some eggs in the game to be able to make more baked goods (lookin at you pumpkin pie), but it requires adding something large like chickens and a chicken coop, and/or some alternative egg sources like growing flax seed and crafting a flax egg.

**Comment by jeremyckahn** [comment]
I'm all for adding chickens and eggs to the game! I want to make sure we get the idea right before moving forward. I've spent some time thinking about this, and I feel pretty strongly about differentiating chicken and cow mechanics.

I think we're on the same page with buying and raising chickens to produce eggs. We'll definitely need chicken breeding, but the goal should be something other than rainbow chickens, because I think that would be a little monotonous for players. So taking that out of the equation, what could we replace it with?

I don't have a clear idea yet, but I have some half-formed thoughts to start off the brainstorming:

  - Higher-level eggs that only result from bred chickens. We could even have multiple tiers of eggs — for example, "1 star eggs" from generations 0-2, "2-star eggs" from generations 3-5, etc.
  - Super Chickens: Some (randomly chosen) purchased chickens naturally produce more eggs per yield than others. Typical chickens would produce one egg at a time, but occasionally some would consistently produce two. The two-egg chickens would (potentially and randomly) produce offspring that may produce three eggs at a time. Three-egg chickens may produce four-egg chickens, and so on infinitely.

I think we could still have color variety among the chickens, but I don't think it needs to be significant (or fantastical) like it is for the cows. I'm not sure if we need a chicken happiness mechanic, but I'm open to it. 

I'd love to hear other's ideas for chicken raising mechanics and rewards!

**Reply by lstebner** [reply]
good stuff! thanks for sharing what you have in mind so far. going off of that.. I do think a happiness mechanic makes sense and should affect the _quality_ of the egg similar to grades of milk. we could grade them C/B/A for consistency, or maybe size them for some variety (small/med/large eggs that the chickens lay as they get older). we cold also introduce a "chick" to "chicken" aging concept that would be a bit different than the cows. 

on breeding.. maybe instead of breeding you can actually incubate the eggs? which adds an 'incubator' item for the shop to sell as well. probably only the highest quality eggs should be able to be incubated. 

chicken feed is another thing that i think needs to exist, and would be similar to cows. maybe you can also choose to feed them seeds, that could be neat, and something i've never seen in another farming game. having some feed in the shop makes sense though.

i like the idea you have about chickens laying more eggs through generations. there's probably some sort of limit to it like 3 or 4, but it made me think what would we do with all those eggs? and that made me think what if each egg can't actually be sold, though they can be used for recipes or incubated, but instead, you had to make a container of a dozen to sell them? that would naturally lead the player to want lots of eggs, and then this mechanic of generational chickens producing lots of eggs fits in nicely. 

i'm cool with not having the color mean anything for breeding and no rainbow chickens here, but it would be nice to just vary them visually. i believe chickens have tons of patterns in the wild, and the eggs they lay usually match their actual color. this would just be nice to see visually if we can incorporate it, and it's realistic! 

lastly, the coop. you didn't mention it, but i just wanted to add that it's probably a thing for sale like the cowpen, and capacity could be upgraded there as well. 

this is going to be awesome!

**Reply by jmmullins** [reply]
Love all that's happening in this thread. All the animals!!

**Comment by jeremyckahn** [comment]
To add on here, I'd also like to have plant-based eggs, but I think we should have chickens with traditional eggs to go along with them. I think players would expect chickens, and it would make the game more interesting!

**Reply by lstebner** [reply]
i think we can make this another discussion. i realize now this one is generically named 'eggs' but it was intended to be about the chicken eggs. i think this will be the first farming game to have a flax egg though whenever we do that!
